app/src/main/res/layout/drawer_loggedin.xml

```diff
@@ -407,32 +407,65 @@
         </LinearLayout>
     </RelativeLayout>
 
-    <LinearLayout
+    <RelativeLayout
+        android:id="@+id/multireddits"

```

Comment:
Take the `@+id/multi` from the LinearLayout here, you can delete the LinearLayout's id since it doesn't need to be referred to

This is important since you can disable individual drawer items from showing up in the settings. It does this by hiding the `multi` id in this case. Here's what it looks like with multireddits disabled without that

![image](https://user-images.githubusercontent.com/1830331/64553985-66eacc00-d332-11e9-9d9c-1c827cfff627.png)

You'll have to change `LinearLayout multiReddits = header.findViewById(R.id.multi);` to be a RelativeLayout in MainActivity also